libnmstate/nm/ovs.py

```diff
@@ -150,15 +170,39 @@ def _get_bridge_port_info(port_profile, devices_info):
     port_slave_names = [c.get_interface_name() for c in port_slave_profiles]
 
     if port_slave_names:
-        iface_slave_name = port_slave_names[0]
-        port_info["name"] = iface_slave_name
+        number_of_interfaces = len(port_slave_names)
+        if number_of_interfaces == 1:
+            port_info[OB.Port.NAME] = port_slave_names[0]
+        else:
+            port_lag_info = _get_lag_info(
+                port_name, port_setting, port_slave_names
+            )
+            port_info.update(port_lag_info)
         if vlan_mode:
             port_info["vlan-mode"] = vlan_mode
             port_info["access-tag"] = port_setting.props.tag
 
     return port_info
 
 
+def _get_lag_info(port_name, port_setting, port_slave_names):
+    port_info = {}
+
+    lacp = port_setting.props.lacp
+    mode = port_setting.props.bond_mode
+    if not mode and lacp:
```

Comment:
Why is `lacp` a boolean here. The previous code makes it look like it would be `off` or `active`.